packages/host/app/components/operator-mode/card-schema-editor.gts

```diff
@@ -184,6 +206,17 @@ export default class CardSchemaEditor extends Component<Signature> {
     ).includes(fieldName);
   }
 
+  get totalOwnField(): number {
+    let total = 0;
```

Comment:
Suggestion to put this into one line: 

`this.args.cardType.fields.filter(field => this.isOwnField(field.name)).length`;